src/test/cypress/support/pageobjects/exercises/quiz/DragAndDropQuiz.ts

```diff
@@ -1,10 +1,37 @@
 import { EXERCISE_BASE, POST } from '../../../constants';
+import { MODELING_EDITOR_CANVAS } from '../modeling/ModelingEditor';
 
 export class DragAndDropQuiz {
     dragItemIntoDragArea(itemIndex: number) {
         cy.get('#drag-item-' + itemIndex).drag('#drop-location');
     }
 
+    setTitle(title: string) {
+        cy.get('#quiz-title').type(title);
+    }
+
+    dragUsingCoordinates(x: number, y: number) {
```

Comment:
I think it would be great if there is a JSDoc here to explain what this method does.